eng/helix/content/InstallNode.ps1

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
Can you add a doc-block? Example:
https://github.com/aspnet/AspNetCore/blob/8bb34bb30bd55cb26b928c05a6a12672bbe5b76a/eng/helix/content/mssql/InstallSqlServerLocalDB.ps1#L1-L14